gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.7-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10-bin.zip
```

Comment:
The `wrapper` task should be updated as well:
```groovy
task wrapper(type: Wrapper) {
    gradleVersion = '4.10'
}
```

Reply:
You're right.